core/modules/qana/RelationGraph.cc

```diff
@@ -668,26 +667,29 @@ void RelationGraph::_fuse(JoinRef::Type joinType,
     _map.fuse(g._map, natural, usingCols);
     // Add spatial edges
     if (!outer && joinSpec && joinSpec->getOn()) {
-        _addSpEdges(joinSpec->getOn(), overlap);
+        _addSpEdges(joinSpec->getOn());
     }
 }
 
 /// This constructor creates a relation graph for a single partitioned
 /// table reference.
-RelationGraph::RelationGraph(TableRef& tr,
-                             TableInfo const* info,
-                             double overlap) :
+/// `matchAngSep` - assumed angular separation for match tables
+RelationGraph::RelationGraph(TableRef& tr, TableInfo const* info) :
     _query(0)
 {
     typedef std::vector<ColumnRefConstPtr>::const_iterator Iter;
 
     if (!info) {
         return;
     } else if (info->kind != TableInfo::MATCH) {
+        LOGS(_log, LOG_LVL_DEBUG, "RG: non-match table tr=\"" << tr << "\" info=" << *info);
         _vertices.push_back(Vertex(tr, info));
         ColumnVertexMap m(_vertices.front());
         _map.swap(m);
     } else {
+        double matchAngSep = static_cast<MatchTableInfo const&>(*info).angSep;
```

Comment:
I think a polymorphic ctor would be cleaner?

Reply:
Agree, I'd like to cleanup TableInfo and try to avoid casts, that is something for future tickets.

Reply:
👍